flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/OperatorSnapshotResult.java

```diff
@@ -30,7 +31,7 @@
  */
 public class OperatorSnapshotResult {
 
-	private RunnableFuture<KeyGroupsStateHandle> keyedStateManagedFuture;
+	private RunnableFuture<KeyedStateHandle> keyedStateManagedFuture;
 	private RunnableFuture<KeyGroupsStateHandle> keyedStateRawFuture;
```

Comment:
I think this generic type can also be changed to the higher-level interface `RunnableFuture<KeyedStateHandle>`.